website/docs/reference/resource-properties/versions.md

```diff
@@ -35,24 +35,29 @@ The standard convention for naming model versions is `<model_name>_v<v>`. This h
 
 The version identifier for a version of a model. This value can be numeric (integer or float), or any string.
 
-The value of the version identifier is used to order versions of a model relative to one another. If a versioned model does _not_ explicitly configure a [`latest_version`](resource-properties/latest-version), the highest version number is used as the latest version to resolve `ref` calls to the model without a `version` argument.
+The value of the version identifier is used to order versions of a model relative to one another. If a versioned model does _not_ explicitly configure a [`latest_version`](resource-properties/latest_version), the highest version number is used as the latest version to resolve `ref` calls to the model without a `version` argument.
 
 In general, we recommend that you use a simple "major versioning" scheme for your models: `v1`, `v2`, `v3`, etc, where each version represents a breaking change from previous versions. However, you are welcome to use other versioning schemes.
```

Comment:
>However, you are welcome to use other versioning schemes 

as long as they behave correctly when `sort()`ed. (or however we're actually doing it). 

On that note, do we handle people putting `v`s in their yaml? What would happen if I did this? 

```
models:
  - name: dim_customers
    versions: 
      - v: v1
      ...
      - v: 2
```
Both from a sorting perspective and an alias-creation perspective - would I wind up with `dim_customers_vv1`?

Reply:
Yes & yes. I'll add an explicit caution that people should not include `v` in their version identifier.